tasks/jsinspect.js

```diff
@@ -24,9 +25,21 @@ module.exports = function(grunt) {
       identifiers: false,
       failOnMatch: true,
       suppress:    100,
-      reporter:    'default'
+      reporter:    'default',
+      configFile:  '.jsinspectrc'
     });
 
+    if (fs.existsSync(options.configFile) && fs.lstatSync(options.configFile).isFile()) {
+      var contents = strip(fs.readFileSync(options.configFile, 'utf8'));
```

Comment:
Also this is a breaking change.
People currently would not be able to update the grunt-plugin without breaking their stack.

I think that's fine - but as said, these are facts that should be clearly communicated. :bowtie: